index.htm

```diff
@@ -22,22 +22,21 @@ <h1 class="h2 pt-3 font-weight-bold">Rewarding Volunteer Distributed Computing</
             <div class="col-12 py-3">
                 <h3 class="h2 font-weight-bold py-3">What is Gridcoin?</h3>
                 <p>
-                    Gridcoin is an open source <a href="https://en.wikipedia.org/wiki/Cryptocurrency">cryptocurrency</a> (Ticker: GRC) which securely rewards <a href="https://en.wikipedia.org/wiki/Volunteer_computing">volunteer computing</a> performed on the <a href="https://boinc.berkeley.edu/">BOINC platform</a>.
+                    Gridcoin is an open source <a href="https://en.wikipedia.org/wiki/Cryptocurrency">cryptocurrency</a> (Ticker: GRC) that rewards <a href="https://en.wikipedia.org/wiki/Volunteer_computing">volunteer computing</a> for science through the <a href="https://boinc.berkeley.edu/">BOINC platform</a>.
                 </p>
                 <h3 class="h2 font-weight-bold py-3">What is BOINC?</h3>
                 <p>
-                    <a href="https://boinc.berkeley.edu/">BOINC</a> is an open source volunteer computing grid which combines the processing power of individual users for the purposes of scientific research. It is free to use and already home to 30+ projects spanning a range of scientific disciplines. Examples include protein structure prediction (<a href="https://boinc.bakerlab.org/rosetta/">Rosetta@home</a>), mapping the Milky Way galaxy (<a href="https://milkyway.cs.rpi.edu/milkyway/">Milkyway@home</a>), and tackling problems in public health and clean energy (<a href="https://www.worldcommunitygrid.org/discover.action">World Community Grid</a>).
+                    <a href="https://boinc.berkeley.edu/">BOINC</a> is an open source platform for volunteer computing that lets individuals use their computers & phones to help science research. It is free to use and already home to 30+ projects over a range of scientific disciplines. Examples include protein structure prediction (<a href="https://boinc.bakerlab.org/rosetta/">Rosetta@home</a>), mapping the Milky Way galaxy (<a href="https://milkyway.cs.rpi.edu/milkyway/">Milkyway@home</a>), and tackling problems in public health and clean energy (<a href="https://www.worldcommunitygrid.org/discover.action">World Community Grid</a>).

```

Comment:
is the 30+ number still true?

Reply:
At least according to the BOINC website:
> About 30 science projects use BOINC. They investigate diseases, study climate change, discover pulsars, and do many other types of scientific research. 

https://boinc.berkeley.edu/


Although I counted ~50 on boinctstats list but not sure how many are in beta, or not running, testing versions of a project, etc. Subtracting a few of those that I could spot brought it down to 45. Probably more that I missed there.